hefquin-engine/src/main/java/se/liu/ida/hefquin/engine/federation/access/impl/FederationAccessManagerBase1.java

```diff
@@ -49,66 +49,63 @@ public abstract class FederationAccessManagerBase1 implements FederationAccessMa
 
 	protected static final Var countVar = Var.alloc("__hefquinCountVar");
 
-	protected AtomicLong issuedCardRequestsSPARQL  = new AtomicLong(0L);
-	protected AtomicLong issuedCardRequestsTPF     = new AtomicLong(0L);
-	protected AtomicLong issuedCardRequestsBRTPF   = new AtomicLong(0L);
+	protected AtomicLong issuedCardRequestsSPARQL = new AtomicLong( 0L );
+	protected AtomicLong issuedCardRequestsTPF =    new AtomicLong( 0L );
+	protected AtomicLong issuedCardRequestsBRTPF =  new AtomicLong( 0L );
 
 	@Override
-	public CompletableFuture<CardinalityResponse> issueCardinalityRequest(
-			final SPARQLRequest req,
-			final SPARQLEndpoint fm )
-					throws FederationAccessException
+	public CompletableFuture<CardinalityResponse> issueCardinalityRequest( final SPARQLRequest req,
+	                                                                       final SPARQLEndpoint fm )
+		throws FederationAccessException
```

Comment:
Please revert this change. I have been trying to keep the code laid out in a way that the lines are not longer than 80 characters, which makes it easier for people with small screens (like me on my laptop, where otherwise I would always have to scroll back and forth horizontally to see the whole code).
```suggestion
	public CompletableFuture<CardinalityResponse> issueCardinalityRequest(
			final SPARQLRequest req,
			final SPARQLEndpoint fm )
					throws FederationAccessException
```

Reply:
Same in the similar cases below.

Reply:
Ah, I see. That's 80 characters for a single line ignoring indentation? I've had my (soft) limit set at 100 (including indentation) based on existing code. I'll revert the code and use 80 chars as the reference for the future.